tests/queries/0_stateless/01721_engine_file_truncate_on_insert.sql

```diff
@@ -0,0 +1,20 @@
+INSERT INTO TABLE FUNCTION file('01718_file/test/data.TSV', 'TSV', 'id UInt32') VALUES ('file', 42);
+ATTACH TABLE test FROM '01718_file/test' (id UInt8) ENGINE=File(TSV);
+
+CREATE TABLE file_engine_table (id UInt32) ENGINE=File(TabSeparated);
```

Comment:
Please add

`IF NOT EXISTS` to CREATE TABLE at the beginning
and `DROP TABLE` at the end of the test.